docs/guides/publish.md

```diff
@@ -0,0 +1,50 @@
+# Publishing a package
+
+uv does not yet have dedicated commands for building and publishing a package. Instead, you can use
+the PyPA tools [`build`](https://github.com/pypa/build) and
+[`twine`](https://github.com/pypa/twine).
```

Comment:
"both of which can be accessed via `uvx`."?

Reply:
```suggestion
[`twine`](https://github.com/pypa/twine), both of which can be invoked via `uvx`.
```